src/submission/submission-error-messages.ts

```diff
@@ -124,13 +124,13 @@ const ERROR_MESSAGES: { [key: string]: (errorData: any) => string } = {
   INVALID_LOST_TO_FOLLOW_UP_ID: ({ info: { lost_to_followup_after_clinical_event_id } }) =>
     `The identifier '${lost_to_followup_after_clinical_event_id}' submitted in the 'lost_to_followup_after_clinical_event_id' field does not exist in your clinical submission.`,
   INVALID_SUBMISSION_AFTER_LOST_TO_FOLLOW_UP: ({
-    info: {
-      lost_to_followup_after_clinical_event_id,
-      interval_of_followup,
-      submitter_treatment_id,
-    },
+    info: { lost_to_followup_after_clinical_event_id, interval_of_followup, treatment_id },
```

Comment:
treatment_id is any of `submitter_treatment_id || submitter_specimen_id || submitter_follow_up_id`